docs/system-admin-guide/custom-fields/README.md

```diff
@@ -84,12 +87,58 @@ To **delete** a custom field, click on the delete icon next to the respective cu
 
 ## Create a multi-select custom field
 
-For work package custom fields of type **List** and **User** you may also select **multi-select custom fields** to select more than one value at once.
+For work package custom fields of type **List**, **User**, **Version** and **Hierarchy** you may also select **multi-select custom fields** to select more than one value at once.
 
-To create a multi-select custom field follow the same steps as you would when [creating a standard custom field](#add-a-new-custom-field). Select format **List, User or Version** and check the option *Allow multi-select*.
+To create a multi-select custom field follow the same steps as you would when [creating a standard custom field](#add-a-new-custom-field). Select format **List**, **User**, **Version** or **Hierarchy **and check the option *Allow multi-select*.
 
-![Sys-admin-multi-select-custom-field](system-admin-allow-multi-select.png)
+![Allow multi-select values for a custom field in OpenProject](system-admin-allow-multi-select.png)
 
 When using multi-select custom fields, you can add as many options as required. The cross icon next to an option will remove it from the selection. The check mark will save your changes.
 
 ![Multi select custom field in a work package](system-guide-custom-field-work-package.png)
+
+## Create a Hierarchy custom field (Enterprise add-on)
+
+> [!NOTE]
+> Custom fields with type Hierarchy are an Enterprise add-on. Click here for more information on the [OpenProject Enterprise edition](https://www.openproject.org/enterprise-edition/).
+
+Hierarchy custom fields allow organizing hierarchical structures in work packages by making use of multi-level select lists. To create a custom field of type Hierarchy follow the same steps as you would when [creating a standard custom field](#add-a-new-custom-field) and select **Hierarchy** format. You can then name the custom field, allow multi-select, specify if it should be a required field or used as a filter, and activate it for all projects. Click the **Save** button to proceed.
+
+
+
+![Create a new custom field of type hierarchy](openproject_system_guide_new_custom_field_new_hierarchy.png)
+
+You will then see the list of all existing custom fields. Click on the name of the custom field you created to edit it, specify possible values and assign it to projects.
```

Comment:
instead of writing `possible values` I'd suggest `selectable items`. Terminology, a value is, when I assign an item to a work package itself. The the referenced item is the value of this work package.